lib/Infer/AliveDriver.cpp

```diff
@@ -777,6 +791,26 @@ souper::AliveDriver::translateDataflowFacts(const souper::Inst* I,
   }
 }
 
+bool
+souper::AliveDriver::translateDemandedBits(const souper::Inst* I,
+                     IR::Function& F,
+                     souper::AliveDriver::Cache& ExprCache) {
+  FunctionBuilder Builder(F);
+
+  auto DemandedBits = IsLHS ? I->DemandedBits : LHS->DemandedBits;
+
+  if (DemandedBits.isAllOnesValue() || DemandedBits.getBitWidth() != I-> Width) {
```

Comment:
this second clause,  what is it doing?

Reply:
Is there any other not-initialized-properly cases?

Reply:
this should definitely be an assertion or an error check, not regular code